src/eda.py

```diff
@@ -88,4 +88,7 @@ def create_and_save_chart(df, col, save_path):
     )
 
     for feature in train_df.columns:
-        create_and_save_chart(train_df, feature, save_path)
+        if feature == 'index' or feature == 'FATALITY':
```

Comment:
Shouldn't need feature=="index" because that feature has been set as the index in line 87

Reply:
not applicable to new code